Course/Course/Presentation/Unit/CourseUnitView.swift

```diff
@@ -296,21 +296,26 @@ public struct CourseUnitView: View {
                             }
                         }
                         VStack {
-                            NavigationBar(
-                                title: isDropdownActive ? sequenceTitle : "",
-                                leftButtonAction: {
-                                    viewModel.router.back()
-                                    playerStateSubject.send(VideoPlayerState.kill)
-                                }).padding(.top, isHorizontal ? 10 : 0)
+                            Group {
+                                NavigationBar(
+                                    title: isDropdownActive ? sequenceTitle : "",
```

Comment:
The screen title is missing when the drop-down isn't active which isn't the desired behavior.  
![image](https://github.com/openedx/openedx-app-ios/assets/5606473/ac2c3dc2-d458-4dcd-b31d-82fb00bf454f)

Reply:
Hi, @saeedbashir, the original upstream `develop` doesn't contain title, so i moved it to under flag. Should i move it to persistent state @volodymyr-chekyrta ?

Reply:
> Hi, @saeedbashir, the original upstream develop doesn't contain title, so i moved it to under flag

If thats the case, you can ignore this request. I thought it got missed because of our recent work.

Reply:
@saeedbashir, now menu is hide when user scrolls content